action.yml

```diff
@@ -22,3 +26,4 @@ runs:
       env:
         INPUT_TOOL: ${{ inputs.tool }}
         INPUT_CHECKSUM: ${{ inputs.checksum }}
+        INPUT_FALLBACK: ${{ inputs.fallback }}
```

Comment:
I'd suggest `INPUT_STRATEGIES` and pass it directly to `cargo-binstall`.

User can specify `crate-meta-data` for trying to download it from official maintainer, `quick-install` from third-party quickinstall, `compile` for from source.

If empty, then disable cargo-binstall

https://github.com/cargo-bins/cargo-binstall/blob/48ee0b0e3e646f7f0cfb2428f46dbcd32afe979b/crates/bin/src/args.rs#L418

Reply:
getting approval for cargo-binstall would be worth attempting IMO if its cargo-quickinstall was disabled.
i.e. trying to explain to my cyber-security team that we're using install-action which falls back to cargo-binstall which falls back to cargo-quickinstall is too hard, and it is that cargo-quickinstall stage which becomes "oh its just grabbing some binary from somewhere on the internet, but they are good people running it, trust them".  The fall back to cargo-binstall is explainable if it is "it builds from source 100%"

Reply:
That's good to hear, disabling quickinstall is pretty easy, you just need to pass `--strategies crate-meta-data,compile` and it will only use the prebuilt from the official repository specified in `Cargo.toml` on https://crates.io , or fallback to `cargo-install`.

`--strategies compile` would then be equivalent to launching `cargo-install`

Reply:
IMO, for a reviewer who is not familiar with the details of cargo-binstall's API but needs to review code that uses this action, something like `strategies: ''`/`strategies: crate-meta-data,compile` is not at all clear at first glance.

Reply:
That's true, maybe we can pass `--disable-strategies` instead.

`--disable-strategies quick-install` is equivalent to `--strategies crate-meta-data,compile`